lib/aws/cognito_srp.rb

```diff
@@ -140,6 +146,53 @@ def refresh_tokens(refresh_token, user_id_for_srp: @user_id_for_srp)
     end
     alias_method :refresh, :refresh_tokens
 
+    def associate_software_token(access_token)
+      @aws_client.associate_software_token(access_token: access_token)
+    end
+
+    def verify_software_token(access_token, user_code, friendly_device_name: nil)
+      params = {
+        access_token: access_token,
+        user_code: user_code,
+        friendly_device_name: friendly_device_name
+      }.compact
+      @aws_client.verify_software_token(params)
+    end
+
+    def set_user_mfa_preference(access_token, software_token_mfa_settings: nil, sms_mfa_settings: nil)
+      params = {
+        access_token: access_token,
+        software_token_mfa_settings: software_token_mfa_settings,
+        sms_mfa_settings: sms_mfa_settings
+      }.compact
+
+      @aws_client.set_user_mfa_preference(params)
+    end
```

Comment:
I think these 3 methods are outside the intended scope of this gem and should be removed. Instead, we can have a section in the documentation explaining how to do this with a regular `Aws::CognitoIdentityProvider::Client` instance.

```suggestion
```

Reply:
I removed 3 methods.